libmscore/fret.cpp

```diff
@@ -293,7 +294,7 @@ void FretDiagram::draw(QPainter* painter) const
             painter->drawLine(QLineF(0.0, y, x2, y));
             }
       QFont scaledFont(font);
-      scaledFont.setPointSizeF(font.pointSize() * _userMag * score()->styleD(Sid::fretMag));
+      scaledFont.setPointSizeF(font.pointSize() * _userMag * score()->styleD(Sid::fretMag) * (spatium() / SPATIUM20));
```

Comment:
It is a mistake to multiply by `score()->styleD(Sid::fretMag)` here and in the other two places in this file where this occurs. I know it is not your mistake, but it is a mistake nonetheless. `score()->styleD(Sid::fretMag)` is simply the style default for `_userMag`, and we are already multiplying by `_userMag`. To see what I mean, increase the "Scale" value for a fret diagram, using the Inspector. Notice that the fret diagram increases in size as it should. Now press the "S" (Set as style) button for the "Scale" property. The fret diagram increases in size again, because we are multiplying by both `score()->styleD(Sid::fretMag)` and `_userMag`.

Reply:
Thank you. I'll admit I am slightly confused about why a lot of the stuff that's written in layout and draw is there. Any more suggestions about how to improve these functions would be very much appreciated :)